drivers/README.md

```diff
@@ -137,5 +130,14 @@ To run the examples, type
       
                 drivers/drive-adv-diff-1D-Cython/drive_adv_diff_1D-setup.py
 
-
-
+6. Directory drive-Lorenz-Delta/ implements the chaotic Lorenz system, with its trademark
+   butterfly shaped attractor. The driver uses the Delta correction feature and Lyapunov
+   estimation to solve for the backward Lyapunov vectors of the system and to accelerate
+   XBraid convergence. Visualize the solution and the Lyapunov vectors with *vis_lorenz_LRDelta.py*
+   Also see example 7 (examples/ex-07.c). *This driver is in a broken state, and needs*
+   *updating for compatibility with new Delta correction implementation.*
```

Comment:
@cadaverous-lives Good to know about this driver.  We can remove this driver if you don't think it's worth fixing, or since it's a driver, it can probably be fixed after the PR.